### ISSUE TITLE
Add new scene in LGSVL 2019.07

### ISSUE THREAD

**Issue by Felixlxc** [lgsvl/simulator#320]
Hi,
    I'd like to add a new scene for testing in the simulator so I follow the instructions of building from source. I use git lfs to download the package and I ended up finding that there is a scene called "loader scene" in the scene folder and scenes such as Borregas Ave are not there. 
    Also I want to know can I just create a new scene in Unity and import all of the assests to generate a new scene and build a webUI for it?
   I'm using a ubuntu16.04 system now. Do I need to use windows for this?

   Thanks a lot.

**Comment by martins-mozeiko** [comment]
We are not distributing BorregaAve assets as unity scene, it is not open-source. We are currently only providing SingleLaneRoad and CubeTown maps as open-source. But we are considering to add more.

You can build maps or vehicles on Linux or Windows. Any OS will work fine.

You can check our [SingleLaneRoad](https://github.com/lgsvl/SingleLaneRoad) or [CubeTown](https://github.com/lgsvl/CubeTown) repositories. These are examples what needs to be done to create map. After you create scene with contents you want it can be build with Simulator -> Build... menu item. This menu item will build AssetBundle from scenes in folders that are placed in Assets/External/Environments folder. So, for example, for CubeTown the build will look for `Assets/External/Environments/CubeTown/CubeTown.unity` scene file. 

Basically if you have NAME map, put it into `Assets/External/Environments/NAME/NAME.unity` location.

For vehicles it is similar - vehicle prefab should be in `Assets/External/Vehicles/NAME/NAME.prefab` location.

Once you build bundles with Simulator->Build menu, you'll be able to add them in WebUI and launch simulation with it. WebUI does not need to be rebuild. It can add new asset bundles from anywhere.(http:// or file:// location).
